src/variables.scss

```diff
@@ -1,6 +1,7 @@
 $brightBlue: #0069BF;
 $darkBlue: #4E677E;
-$lightGray: #FFFFFF;
+$lightGray:#F9FAFB;
```

Comment:
The gray background color for the header is slightly different than the one for the top of the page. Header is #F9FAFB while the top of the page that includes the breadcrumb and title should be #F8F9FC.